modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -411,3 +411,21 @@ test('LPD-33048 Last login visibility', async ({usersAndOrganizationsPage}) => {
 		usersAndOrganizationsPage.tableOrderLastLoginDateItem
 	).toBeVisible();
 });
+
+test('LPD-35562 Enter reserved screen name', async ({addUserPage}) => {
+	await addUserPage.goToAddUser();
+
+	await addUserPage.addUser();
```

Comment:
This is testing the wrong thing. You need to add the screen name to the reserved credential in instance settings:

![Screenshot 2024-10-02 at 2 14 57 PM](https://github.com/user-attachments/assets/4d2aecd3-f2c7-4fa3-832d-2b83229b667b)

Reply:
I have no comment.